common/src/main/cpp/extensions/openxr_fb_body_tracking_extension_wrapper.cpp

```diff
@@ -0,0 +1,359 @@
+	{ XRBodyTracker::JOINT_RIGHT_MIDDLE_FINGER_PHALANX_DISTAL, XR_BODY_JOINT_RIGHT_HAND_MIDDLE_DISTAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_MIDDLE_FINGER_TIP, XR_BODY_JOINT_RIGHT_HAND_MIDDLE_TIP_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_RING_FINGER_METACARPAL, XR_BODY_JOINT_RIGHT_HAND_RING_METACARPAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_RING_FINGER_PHALANX_PROXIMAL, XR_BODY_JOINT_RIGHT_HAND_RING_PROXIMAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_RING_FINGER_PHALANX_INTERMEDIATE, XR_BODY_JOINT_RIGHT_HAND_RING_INTERMEDIATE_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_RING_FINGER_PHALANX_DISTAL, XR_BODY_JOINT_RIGHT_HAND_RING_DISTAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_RING_FINGER_TIP, XR_BODY_JOINT_RIGHT_HAND_RING_TIP_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_PINKY_FINGER_METACARPAL, XR_BODY_JOINT_RIGHT_HAND_LITTLE_METACARPAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_PINKY_FINGER_PHALANX_PROXIMAL, XR_BODY_JOINT_RIGHT_HAND_LITTLE_PROXIMAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_PINKY_FINGER_PHALANX_INTERMEDIATE, XR_BODY_JOINT_RIGHT_HAND_LITTLE_INTERMEDIATE_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_PINKY_FINGER_PHALANX_DISTAL, XR_BODY_JOINT_RIGHT_HAND_LITTLE_DISTAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_PINKY_FINGER_TIP, XR_BODY_JOINT_RIGHT_HAND_LITTLE_TIP_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+};
+
+OpenXRFbBodyTrackingExtensionWrapper *OpenXRFbBodyTrackingExtensionWrapper::singleton = nullptr;
+
+OpenXRFbBodyTrackingExtensionWrapper *OpenXRFbBodyTrackingExtensionWrapper::get_singleton() {
+	if (singleton == nullptr) {
+		singleton = memnew(OpenXRFbBodyTrackingExtensionWrapper());
+	}
+	return singleton;
+}
+
+OpenXRFbBodyTrackingExtensionWrapper::OpenXRFbBodyTrackingExtensionWrapper() :
+		OpenXRExtensionWrapperExtension() {
+	ERR_FAIL_COND_MSG(singleton != nullptr, "An OpenXRFbBodyTrackingExtensionWrapper singleton already exists.");
+
+	request_extensions[XR_FB_BODY_TRACKING_EXTENSION_NAME] = &fb_body_tracking_ext;
+
+	singleton = this;
+}
+
+OpenXRFbBodyTrackingExtensionWrapper::~OpenXRFbBodyTrackingExtensionWrapper() {
+	cleanup();
+}
+
+void OpenXRFbBodyTrackingExtensionWrapper::_bind_methods() {
+}
+
+void OpenXRFbBodyTrackingExtensionWrapper::cleanup() {
+	fb_body_tracking_ext = false;
+}
+
+uint64_t OpenXRFbBodyTrackingExtensionWrapper::_set_system_properties_and_get_next_pointer(void *next_pointer) {
+	system_body_tracking_properties.type = XR_TYPE_SYSTEM_BODY_TRACKING_PROPERTIES_FB;
+	system_body_tracking_properties.next = next_pointer;
+	system_body_tracking_properties.supportsBodyTracking = false;
```

Comment:
We're standardizing on using braced-initialization for enums initialization. See https://github.com/GodotVR/godot_openxr_vendors/pull/100#issuecomment-1981726339 for reference.

Reply:
Switched to brace initialization.